### PR TITLE
Dynamic name on results pane

### PR DESCRIPTION
**Pull request by justinmc**
Previously, the results page has 1 field that is hard coded to show "OUTPUT ENERGY".  This PR will make that title dynamic.

### CODE REVIEW

File: client/public/js/components/status.jsx [hunk 1]
```diff
@@ -93,16 +93,18 @@ function Status(props) {
...
     let resultUnit;
+    let resultValue;
 
     if (outputResultsIndex !== -1) {
       const outputResults = props.workflow.run.outputs.get(outputResultsIndex)
         .fetchedValue;
 
       if (outputResults.singlet_energy) {
-        resultValue = outputResults.singlet_energy.value;
+        resultName = outputResults.singlet_energy.name;
```
Comment (justinmc) [inline client/public/js/components/status.jsx]: Does this look doable for you to send in the outputs to step1?  Do we want to take this one step further and make there be a dynamic number of output values/names/units to display?

Comment (avirshup) [inline client/public/js/components/status.jsx]: Yes and yes!

**Comment by justinmc** [comment]
Should receive something like this for the output values:

   output_values: [
    {
      value: '1234',
      unit: 'hartree',
      name: 'Output Energy'
  ]

**Comment by justinmc** [comment]
Got the 👍 from Aaron via slack.

**Comment by justinmc** [comment]
Ticked off a box in issue https://github.com/Autodesk/molecular-design-applications/issues/76

**Pull request merged**